A32NX/html_ui/Pages/A32NX_Core/A32NX_ATSU.js

```diff
@@ -134,12 +134,12 @@ const lbsToKg = (value) => {
  * @param {() => void} updateView
  */
 const getSimBriefOfp = (mcdu, updateView) => {
-    const simBriefUsername = NXDataStore.get("CONFIG_SIMBRIEF_USERNAME", "");
+    const simBriefUsername = NXDataStore.get("CONFIG_SIMBRIEF_USERNAME", ""); // TODO legacy, remove once most users have transitioned away
     const simBriefUserId = NXDataStore.get("CONFIG_SIMBRIEF_USERID", "");
 
     if (!simBriefUsername && !simBriefUserId) {
         mcdu.addNewMessage(NXFictionalMessages.noSimBriefUser);
-        throw ("No simbrief username/user ID provided");
+        throw ("No simbrief pilot ID provided");
```

Comment:
```suggestion
        throw ("No SimBrief pilot ID provided");
```
This currently doesn't match a similar error on another line:
https://github.com/flybywiresim/a32nx/blob/fb302ad852cba138c43630e15ac9ba1e5ca09977/A32NX/html_ui/Pages/A32NX_Utils/SimBriefApi.js#L9